package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Backend service for the SE2 Project",
   "main": "main.js",
   "scripts": {
-    "start": "node main.js",
+    "start": "npm i && node main.js",
```

Comment:
This is not best practice and we should probably remove it. If you really need that for the `docker-compose.yml`, please create a second script that executes `npm install` and `npm run start`.